### ISSUE TITLE
Embedded Postgresql refuses to start

### ISSUE THREAD

**Issue by NassimBounouas** [alfio-event/alf.io#557]
**Describe the bug**

For few days I'm working on alf.io development without any problems and since this evening I can't start embedded postgresql. I tried to reclone the project and it's the same result : 

````
gradlew startEmbeddedPgSQL

Welcome to Gradle 4.10.2!

Here are the highlights of this release:
 - Incremental Java compilation by default
 - Periodic Gradle caches cleanup
 - Gradle Kotlin DSL 1.0-RC6
 - Nested included builds
 - SNAPSHOT plugin versions in the `plugins {}` block

For more details see https://docs.gradle.org/4.10.2/release-notes.html

Starting a Gradle Daemon, 1 stopped Daemon could not be reused, use --status for details

> Task :startEmbeddedPgSQL FAILED
Failed to extract file set
java.lang.NullPointerException: executable is NULL
[...]
FAILURE: Build failed with an exception.

* Where:
Build file 'C:\####\####\####\alf.io\build.gradle' line: 336

* What went wrong:
Execution failed for task ':startEmbeddedPgSQL'.
> java.lang.NullPointerException (no error message)

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 7s
1 actionable task: 1 executed
````

Has someone experienced that problem ? 

Thanks

**Comment by syjer** [comment]
Hi @NassimBounouas , could you try to set a specific version to the following plugin: 

 `classpath 'ru.yandex.qatools.embed:postgresql-embedded:+'`

For example:

 `classpath 'ru.yandex.qatools.embed:postgresql-embedded:2.9'`

 (you can try the various versions at:

https://search.maven.org/search?q=g:ru.yandex.qatools.embed%20AND%20a:postgresql-embedded&core=gav

).

Alternatively, it's to remove the caches from the plugin (which I don't know where it unpack the files under windows :( )

**Comment by syjer** [comment]
thank you for checking @NassimBounouas :). Yes, we will fix the version to 2.9

**Comment by syjer** [comment]
done, thank you @NassimBounouas  :+1:
